app/jinja_filters.py

```diff
@@ -565,6 +565,7 @@ def map_list_collector_config(
 
     for list_item in list_items:
         item_name = list_item.get("item_title")
+        item_index = list_items.index(list_item) + 1
```

Comment:
This could be:
```py
    for index, list_item in enumerate(list_items, start=1):
        item_name = list_item.get("item_title")
```

Reply:
So would you then use `index` instead of `item_index` to set the data-qa?